docs/specification/STU2.md

```diff
@@ -598,6 +598,57 @@ Content-Type: application/json
 }
 ```
 
+## Conformance
+The FHIRcast specification can be described as a set of capabilities and any specific FHIRcast Hub may implement a subset of these capabilities. A FHIRcast Hub declares support for FHIRcast and specific capabilities by exposing an extension on its FHIR server's CapabilityStatement as described below. 
+
+### Declaring support for FHIRcast 
+A server declares support for FHIRcast using the FHIRcast extension on its FHIR CapabilityStatement's `rest` element. The FHIRcast extension has the following internal components:
+
+Component | Cardinality | Type | Description
+--- | --- | --- | ---
+`eventsSupported` | 1..* | string | Space-delimited list of FHIRcast events supported by the Hub.
+`hub.url`| 0..1 | url | The url at which an app subscribes. May not be supported by client-side Hubs.
+`websocketSupport` | 1..1 | boolean | The static value: `true`, indicating support for websockets.
```

Comment:
Can this be false? If not then what is the point of having it?

Reply:
well, yeah, that's a good question. Per the spec, `websocketSupport` is required to be true, but I was anticipating maybe STU1 implementations or non-compliant Hubs ....